app/models/message.rb

```diff
@@ -1,4 +1,6 @@
 class Message < ApplicationRecord
+  belongs_to :from, class_name: 'User'
+  belongs_to :to, class_name: 'User'
   belongs_to :chat
   validates :chat, presence: true
```

Comment:
I think you should validate the presence of `from` and `to`.

Reply:
It's done by default in rails 5 (with `belongs_to`)

Reply:
Yes, but we tested it on @jpascual1994 console but it's not working. Maybe there is some configuration missing or depends on the Rails 5 version?

Reply:
@MaicolBen confirmed. He must set `Rails.application.config.active_record.belongs_to_required_by_default = true` in `config/initializers/new_framework_defaults.rb`